apps/web/src/components/templates/TemplateFormProvider.tsx

```diff
@@ -15,6 +16,18 @@ export const TemplateFormProvider = ({ children }) => {
         errors.notificationGroup = 'Required field notification group';
       }
 
+      data.steps.map((step, index) => {
```

Comment:
Small one could use .forEach here :)

Reply:
Right right, forgot to go back and do that. Thanks!